src/shipping/shipping-in-store-delivery.md

```diff
@@ -0,0 +1,44 @@
+---
+title: In-store Delivery
+---
+
+The In-store delivery method allows to select a source to be used as a pickup location during the checkout of your order.
```

Comment:
```suggestion
With the in-store delivery method,  you can select a source to be used as a pickup location during the checkout.
```

Reply:
done!, and I agree with you Kevin, this is something we can raise with someone? (ENG or Product?), and then we can work on applying necessary changes?. Count me in!